File: DSA_CONCEPTS/Arrays_representation/prob.js
const arr = [4, 2, 1, 3];
arr.sort();
console.log(arr);

let res = 1;
for (let i = 1; i < arr.length; i++) {
  let sum = 0;
  for(let j=0;j<i;j++){
    sum +=arr[j]
  }
  if(arr[i]>=sum){
    res++;
  }
}

// arr.forEach((num,i)=>{
//     if(i===0){
//         res++;
//     }
//     let sum=0;
//     for(let j=0;j<i;j++){
//         sum=arr[j];
//     }
//     if(num>=sum){
//         res++;
//     }

// })
console.log("ANSWER", res);

File: DSA_CONCEPTS/Arrays_representation/prob.py
A = [4, 2, 1, 3]
N = 4
A.sort()
res = 1
for i in range(1,N):
    sum = 0
    for j in range(0,i):
        sum += A[j]
    if(A[i]>=sum):
        res += 1
print(res)